buildroot-external/package/hassio/fetch-container-image.sh

```diff
@@ -11,6 +11,26 @@ image_json_name=$4
 dl_dir=$5
 dst_dir=$6
 
+retry() {
+	local retries="$1"
+	local cmd=$2
+
+	local output
+	output=$(eval "$cmd")
+	local rc=$?
+
+	# shellcheck disable=SC2086
+	if [ $rc -ne 0 ] && [ $retries -gt 0 ]; then
+		echo "Retrying \"$cmd\" $retries more times..." >&2
```

Comment:
Currently `retry 3 ...` actually does N **retries**, not sure whether it shouldn't do N-1, so the actual number of tries would be what was passed as the first argument. Let me know what you think.

Reply:
Hm, yeah N-1 seems what I would have expected. But then, it's retry logic, the number is a bit arbitrary anyways. I don't care that much.